### ISSUE TITLE
Plugin.Firebase.iOS cannot be resolved

### ISSUE THREAD

**Issue by maxbartunik** [TobiasBuchholz/Plugin.Firebase#125]
I am attempting to use Plugin.Firebase in a MAUI project. However, after install (either nuget or #65) Plugin.Firebase.iOS does not seem to exist. Android, Auth, etc. resolve correctly.

In my MAUIProgram.cs:
```
#if IOS
using Plugin.Firebase.iOS; //<-- does not exist
#else
using Plugin.Firebase.Android;
#endif
```
"The type or namespace name 'iOS' does not exist in the namespace 'Plugin.Firebase' (are you missing an assembly reference?)"

I am using VS 17.4.4 on Windows.

I would appreciate any hints as to what I am missing...

**Comment by nabilakhlaque** [comment]
Why is this closed? This is still an issue.

**Comment by FerArreguin92** [comment]
Does anyone have any news regarding this issue? I'm using net6.0 and have this issue.

**Comment by sej69** [comment]
I think the path should be:
using Plugin.Firebase.Bundled.Platforms.iOS;

However, even though it's showing as resolved in code (in the using statement), I get a:
The type or namespace iOS does not exist in the namespace 'Plugin.Firebase.Bundled.Platforms'

**Comment by TobiasBuchholz** [comment]
Yeah tha'ts correct, with version 2.0.0 the namespace got cleaned up and changed a bit. Please provide an example project so I can look into it. Also what OS and Visual Studio version are you using?

**Comment by K1LL3R234** [comment]
#if IOS
using Plugin.Firebase.Core.Platforms.iOS;
#else
using Plugin.Firebase.Core.Platforms.Android;
#endif

**Comment by K1LL3R234** [comment]
`#if IOS
using UIKit;
using Plugin.Firebase.Core.Platforms.iOS;
#else
using Plugin.Firebase.Core.Platforms.Android;
#endif`

**Comment by K1LL3R234** [comment]
It should be this

`
#if IOS
using UIKit;
using Plugin.Firebase.Core.Platforms.iOS;
#else
using Plugin.Firebase.Core.Platforms.Android;
#endif
`

**Comment by maxbartunik** [comment]
This issue is fixed for me - I was not able to fix the existing MAUI project, however (one or all of) following steps worked:
- Performed a Visual Studio repair (Visual Studio Installer)
- Cleared NuGet caches (https://learn.microsoft.com/en-us/nuget/consume-packages/managing-the-global-packages-and-cache-folders#clearing-local-folders)
- Created a new .Net MAUI solution
- Add Plugin.Firebase through #65 
- Copy existing code to new solution

Things that did not help:
- Deleted obj and bin folders
- Removed and Re-added Plugin.Firebase through NuGet

**Comment by TobiasBuchholz** [comment]
Since I haven't heard from you in more than 30 days, I hope this issue is no longer affecting you. If it is, please reopen this issue and provide the requested information so that I can look into it further. Thank you!
